LSP-lemminx.sublime-settings

```diff
@@ -36,7 +37,7 @@
                 }
 			],
 			"server": {
-				"workDir": "~/.lemminx"
+				"workDir": "$home/.lemminx"
```

Comment:
It seems inconsistent to have `$foo` here but `${foo}` in other places. Are both allowed? I think only the variant with curly braces should be allowed as then there is less chance of unwanted replaces taking place. And it would be consistent with what ST allows in build configs for example.

Reply:
IIRC, `sublime.expand_variables()` allows both notations